canonical/ubuntu.com
# WD-16349 Dev Rebrand /ceph/what-is-ceph

## Done

- Dev Rebrand /ceph/what-is-ceph
- No change in content.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Design](https://www.figma.com/design/ulUyOFdQ7CNy0whuJ4S2FM/Ceph-Bubble?node-id=10-2370&node-type=instance&m=dev)
- [Demo]()

## Issue / Card

Fixes # [WD-16349](https://warthogs.atlassian.net/browse/WD-16349)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
